### PR TITLE
pages: add "how to preact" document

### PR DESCRIPTION
**Pull request by JoviDeCroock**
This adds a short introduction on how to add a [`wmr site with Preact`](https://github.com/preactjs/wmr).

### CODE REVIEW

File: products/pages/src/content/how-to/deploy-a-wmr-site.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Deploy a WMR site
+
+WMR is a tiny all-in-one development tool for modern web applications, it uses the modern esmodules and uses Preact
+out of the box.
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-wmr-site.md]:
```suggestion
[WMR](https://github.com/preactjs/wmr) is a tiny all-in-one development tool for modern web applications.
It uses the ES Modules and [Preact](https://preactjs.com) out of the box, and pre-renders your pages to static HTML at build time.
```

File: products/pages/src/content/how-to/deploy-a-wmr-site.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+In this guide, you'll create a new Preact application and deploy it using Cloudflare Pages. We'll be using `create-wmr`, a tool to generate a new WMR application.
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-wmr-site.md]:
```suggestion
In this guide, you'll create a new Preact application and deploy it using Cloudflare Pages.
We'll be using [create-wmr](https://www.npmjs.com/package/create-wmr) to quickly generate a new WMR application.
```

File: products/pages/src/content/how-to/deploy-a-wmr-site.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Setting up a new project
+
+Create a new project using `npx`, giving it a title.
+
+```sh
+$ npm init wmr your-project-name
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-wmr-site.md]:
```suggestion
$ npm init wmr your-project-name
$ cd your-project-name
```

File: products/pages/src/content/how-to/deploy-a-wmr-site.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## Creating a GitHub repository
+
+Create a new GitHub repository by visiting [repo.new](https://repo.new). Once you've created a new repository, you can push your local application to GitHub:
+
+```sh
+$ git init
+$ git remote add origin https://github.com/yourgithubusername/githubrepo
+$ git checkout -b main
+$ git add .
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-wmr-site.md]: (possibly for non-windows)

```suggestion
$ git add -A
```

File: products/pages/src/content/how-to/deploy-a-preact-wmr-site.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Deploy a Preact WMR site
+
+[WMR](https://github.com/preactjs/wmr) is a tiny all-in-one development tool for modern web applications.
+It uses the ES Modules and [Preact](https://preactjs.com) out of the box, and pre-renders your pages to static HTML at build time.
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-preact-wmr-site.md]:
```suggestion
It uses ES Modules and [Preact](https://preactjs.com) out of the box, and pre-renders your pages to static HTML at build time.
```

**Review by developit — APPROVED** [review]

**Comment by lukeed** [comment]
Hi @JoviDeCroock and @developit, I made some changes that hopefully put the emphasis more on Preact than WMR. If you can look this over & lemme know if there's anything that needs changing, please let me know :)

File: products/pages/src/content/how-to/deploy-a-preact-site.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+# Deploy a Preact site
+
+[Preact](https://preactjs.com) is a popular, open-source framework for building modern web applications. Preact can also be used as a lightweight alternative to React because the two share the same API and component model.
+
+In this guide, you'll create a new Preact application and deploy it using Cloudflare Pages.
+We'll be using [`wmr`](https://github.com/preactjs/wmr), an all-in-one development tool built by the Preact team, to quickly generate a new WMR application.
```
Comment (developit) [inline products/pages/src/content/how-to/deploy-a-preact-site.md]:
```suggestion
We'll be using [`wmr`](https://github.com/preactjs/wmr), an all-in-one development tool built by the Preact team, to quickly generate an optimized web application.
```

**Review by developit — APPROVED** [review]

**Review by lukeed — APPROVED** [review]

**Review by kristianfreeman — APPROVED** [review]

**Pull request merged**